README.md

```diff
@@ -581,22 +612,24 @@ relevant if you are rendering a complicated view that takes
 significant computational time to generate.
 
 ## Virtual Node
+
 **Properties**
-- [sel](#sel--string)
-- [data](#data--object)
-- [children](#children--array)
-- [text](#text--string)
-- [elm](#elm--element)
-- [key](#key--string--number)
```

Comment:
Since remark re-formats, this is another decision where I had to choose between re-formatting this list to use `*` or a longer list to use `-` I went with the shorter change.